### ISSUE TITLE
GVRUnlitFBOShader renders upside down

### ISSUE THREAD

**Issue by SteveGreatApe** [Samsung/GearVRf#1705]
If I switch my material to use GVRUnlitFBOShader then the texture is rendered upside down, is this deliberate? As I can see it's using a Vertex Template called unlit_flip_tex.fsh, which implies it's supposed to be flipping the texture upside down.

**Comment by NolaDonato** [comment]
This shader used to be used for screen capture. In this case, you want the texture to be flipped so that when you save it in a file it looks correct. We don't use this for screen capture anymore, it is no longer referenced anywhere in our demos repo. We should probably deprecate it.

If you tell me what you are trying to do, I can suggest another shader which will do that for you.

**Comment by SteveGreatApe** [comment]
I just want something to render a texture ignoring the lighting. I originally tried to set the "emissiveTexture" field of GVRPhongShader but I couldn't get that to work properly, not sure if my problem or a bug in the shader. I didn't try too hard to get it working as I figured it was probably more efficient anyway to use a simpler shader that just ignored the lighting.

For the normal lit textures I tried using GVRTextureShader, but couldn't get them to work with ambient lighting, I tried setting u_texture instead of ambientTexture but still found the texture just came out blank under an ambient light, so switched to GVRPhongShader which works nicely.
